syntax/std_strconv.go

```diff
@@ -0,0 +1,22 @@
+package syntax
+
+import (
+	"github.com/arr-ai/arrai/rel"
+)
+
+func stdStrconv() rel.Attr {
+	return rel.NewAttr("strconv", rel.NewTuple(
```

Comment:
It doesn't feel quite right to have these in `strconv`. I've suggested elsewhere. that `eval` should be `eval.value` and `unsafe_eval` should be `eval.expr`. I think it therefore makes sense for `eval` to be the top level package.